.github/workflows/link-pr-to-source-issue.yml

```
@@ -0,0 +1,123 @@
+name: Link PR to Source Issue
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)